Web_Development/Javascript/Topics/2.History/History_Of_JavaScript.md

```diff
@@ -0,0 +1,50 @@
+<br>
+
+# WHAT IS JAVASCRIPT?
+
+<br>
+
+![image](js-1.png)
+<br><br>
+
+# TIMELINE OF JAVASCRIPT
```

Comment:
Use "Title Case" instead of "UPPER CASE" for the headers! That makes it look better.